ui/src/app/features/face-services/face-recognition/recognition-result/recognition-result.component.html

```diff
@@ -39,7 +40,7 @@
       [picture]="resizePhoto"
       [printData]="recalculatePrintData"
       [type]="type"
-      [isLoaded]="isLoaded"
+      [isLoaded]="!pending"
```

Comment:
`isLoaded` is not needed anymore? can it be removed then from the container?

Reply:
We use this in line 50